### PR TITLE
Add @rikinpatel

### PR DESCRIPTION
**Pull request by patelrikin**
This PR is after listening to Syntax episode today

Added uses section on my website. I dont have dedicated page as its a single page layout but you can see in the footer section stuff that I use including website techstack that it was made of.

### CODE REVIEW

**Comment by patelrikin** [comment]
@HugoDF close if its not good enough for the criteria that you guys are looking as eligibility for PR to be merged in.

**Comment by sam-cross** [comment]
Could you make your uses a little clearer on the site? Being in the footer makes it difficult to find.

**Comment by patelrikin** [comment]
@sam-cross made prominent to stand out, let me know what you think.

**Comment by patelrikin** [comment]
@HugoDF can you merge in?

File: src/data.js [hunk 1]
```diff
@@ -1117,6 +1117,27 @@ module.exports = [
       'Golang',
     ],
   },
+  {
+    name: 'Rikin Patel',
+    description: 'Experienced Front-end developer, Passionate about Javascript',
+    url: 'https://patelrikin.com/uses',
```
Comment (HugoDF) [inline src/data.js]: Why don't we make this https://patelrikin.com/#uses

Comment (patelrikin) [reply src/data.js]: I wasnt sure if it would pass validation

Comment (patelrikin) [reply src/data.js]: @HugoDF made changes

**Pull request merged**